testing/eigen/APKBUILD

```diff
@@ -0,0 +1,27 @@
+# Contributor: Bradley J Chambers <brad.chambers@gmail.com>
+# Maintainer: Bradley J Chambers <brad.chambers@gmail.com>
+pkgname=eigen
+pkgver=3.3.4
+pkgrel=0
+pkgdesc="Eigen is a C++ template library for linear algebra"
+url="http://eigen.tuxfamily.org/index.php?title=Main_Page"
+arch="all"
+license="MPL2"
+depends=""
+makedepends=""
+install=""
+subpackages="$pkgname-dev"
+source="$pkgname-$pkgver.tar.gz::http://bitbucket.org/eigen/$pkgname/get/$pkgver.tar.gz"
+builddir="$srcdir/$pkgname-eigen-5a0156e40feb"
```

Comment:
This hash in directory name is not good, it’d complicate bumping. You can workaround it using e.g.:

```sh
builddir="$srcdir/$pkgname-$pkgname"

prepare() {
	mv "$builddir"-* "$builddir"  # directory name contains hash
	default_prepare
}
```